ImpeccableHQ/swapr-dapp
# feature: add click and pro mode events

# Summary

Fixes #1674

This PR adds events for tracking clicking off and pro chart options and pro mode every 15 seconds.

  # To Test

1. Click in pro and off mode
- [ ] Check if events are triggered
2. Go to pro mode and stay on the page for more than 15 seconds
- [ ] Check if pro mode 15 seconds event is triggered with associated chain name
